src/test/java/ome/codecs/LZ4CodecTest.java

```diff
@@ -0,0 +1,49 @@
+package ome.codecs;
```

Comment:
The copyright statement used in other files in this repository should be added to the top of this file.